modules/gapi/include/opencv2/gapi/fluid/gfluidkernel.hpp

```diff
@@ -436,4 +438,28 @@ class GFluidKernelImpl : public cv::detail::KernelTag
 
 } // namespace cv
 
+namespace cv {
+namespace gapi {
+namespace s11n {
+namespace detail {
+    template<> struct S11N<cv::GFluidOutputRois>
+    {
+        static void serialize(IOStream &os, const cv::GFluidOutputRois &r)
+        {
+            os << r.rois;
+        }
+
+        static cv::GFluidOutputRois deserialize(IIStream &is)
+        {
+            cv::GFluidOutputRois r;
+            is >> r.rois;
+            return r;
+        }
+    };
+} // namespace detail
+} // namespace s11n
+} // namespace gapi
+} // namespace cv
```

Comment:
With the test for it?

Reply:
You can put your S11N with your test.

Reply:
I can't because another test cpp-s which use GFluidOutputRois will use instantiations of primary S11N template and not my specialization, and compiler will choose instantiations from my and theirs (in other tests) on its own

Reply:
Should I add my specialization in any test cpp which uses GFluidOutputRois ?

Reply:
No, why? Only where you want to test it for the S11N.

Reply:
Create a JIRA ticket as discussed: https://jira.devtools.intel.com/browse/ADE-2179